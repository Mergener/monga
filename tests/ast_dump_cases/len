function main() {
    @len(new int[5]);
}