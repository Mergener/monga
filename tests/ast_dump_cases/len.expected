<Module>
	<FunctionDefinition>
		<Name>main</Name>
		<ReturnType>void</ReturnType>
		<Block>
			<Echo>
				<Len>
					<New>
						<Type>int</Type>
						<ArraySize>
							<IntConstant>5</IntConstant>
						</ArraySize>
					</New>
				</Len>
			</Echo>
		</Block>
	</FunctionDefinition>
</Module>
